projects/ngx-datatable/src/lib/components/datatable.component.scss

```diff
@@ -39,7 +37,7 @@
   &.scroll-horz {
     .datatable-body {
       overflow-x: auto;
-      -webkit-overflow-scrolling: touch;
+      overflow-scrolling: touch;
```

Comment:
`overflow-scrolling` doesn't exist as a CSS property, see https://caniuse.com/?search=overflow-scrolling and https://css-tricks.com/elastic-overflow-scrolling/. It's a pure iOS feature to disable springy scrolling: https://css-tricks.com/snippets/css/momentum-scrolling-on-ios-overflow-elements/.

IMO, we should keep it as-is (probably even add an inline comment), since changing it will probably break the scrolling behavior under iOS since we do the whole scrolling calculation our self and iOS springy scroll behavior most likely breaks this.

Reply:
adjusted